src/main/java/mate/academy/rickandmorty/controller/CharacterController.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.CharacterDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Character management", description = "Endpoints for getting characters from DB")
+@RequiredArgsConstructor
+@RestController
+public class CharacterController {
+    private final CharacterService characterService;
+
+    @Operation(summary = "Get random character",
+            description = "Get a random characters from Rick and Morty world")
+    @GetMapping("/random")
```

Comment:
REST URI convention violates here.
```suggestion
    @GetMapping
```